models/Document.php

```diff
@@ -752,7 +752,7 @@ public function hasSiblings($includingUnpublished = null)
     /**
      * Returns true if the element is locked
```

Comment:
Guess this comment is wrong then. IMHO, it returns one of the enum values defined in tree_locks table
(i.e. 'self','propagate')

Reply:
I added the enom values to all setLocked/getLocked functions. Ok?

Reply:
👍